cdap-cli/src/main/java/co/cask/cdap/cli/ArgumentName.java

```diff
@@ -139,4 +139,25 @@ public String getName() {
   public String toString() {
     return name;
   }
+
+  public static final String ENTITY_DESCRIPTION_STRING = String.format("'<%s>' " +
+    "is of the form '<entity-type>:<entity-id>', where '<entity-type>' is one of " +
+    "'artifact', 'app', 'dataset', 'program', 'stream', or 'view'.\n" +
```

Comment:
Not that I am aware of. I though this was limited to these entity-types, as described in the Ref Manual http://docs.cask.co/cdap/current/en/reference-manual/http-restful-api/metadata.html#metadata-properties